### PR TITLE
change model

### PR DESCRIPTION
**Pull request by MohammadKoolivand**
User model has been changed for new signup strategy

### CODE REVIEW

**Review by amirhnr — APPROVED** [review]

**Review by amirhnr — APPROVED** [review]

**Review by alikia1995 — APPROVED** [review]
dear Mohammad.
tanks for your hard work. I verify your code

File: src/models/User.js [hunk 1]
```diff
@@ -4,8 +4,8 @@ const Schema = mongoose.Schema;
 
 const UserSchema = new Schema(
   {
-    username: { type: String, required: true, unique: true, index: true },
-    password: { type: String, required: true },
+    username: { type: String, unique: true, index: true },
+    password: { type: String },
     mobile: { type: String, required: true, unique: true },
```
Comment (farshid0594) [inline src/models/User.js]: add index:true to mobile

**Review by farshid0594 — CHANGES_REQUESTED** [review]
add index to mobile field

**Review by mehdigadyari — APPROVED** [review]

**Review by alikia1995 — APPROVED** [review]

**Review by alikia1995 — APPROVED** [review]

File: src/models/User.js (hunk 1 above)
Comment (MohammadKoolivand) [reply src/models/User.js]: done

**Review by farshid0594 — APPROVED** [review]

**Pull request merged**